lib/chef/provisioning/machine/unix_machine.rb

```diff
@@ -223,6 +223,10 @@ def detect_sh
 platform="aix"
 platform_version=`uname -v`
 machine="ppc"
+# Linux supporting /etc/os-release
```

Comment:
@glennmatthews Would you mind opening a PR against https://github.com/chef/opscode-omnitruck/blob/master/views/install.sh.erb to also support this?  It would also be nice to put a comment at the top of this big String with a link to the `install.sh.erb` we are trying to stay in sync with so future users don't have this same issue.